conformance/client/src/main/kotlin/com/connectrpc/conformance/client/OkHttpEventTracer.kt

```diff
@@ -0,0 +1,83 @@
+// Copyright 2022-2023 The Connect Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.connectrpc.conformance.client
+
+import okhttp3.Call
+import okhttp3.Connection
+import okhttp3.EventListener
+import okhttp3.Protocol
+import okhttp3.Request
+import okhttp3.Response
+import java.io.IOException
+import java.net.InetSocketAddress
+import java.net.Proxy
+
+internal class OkHttpEventTracer(
+    private val printer: VerbosePrinter.Printer,
+) : EventListener() {
+    override fun connectStart(call: Call, inetSocketAddress: InetSocketAddress, proxy: Proxy) {
+        printer.printlnWithStackTrace("connecting to $inetSocketAddress...")
+    }
+    override fun connectEnd(
+        call: Call,
+        inetSocketAddress: InetSocketAddress,
+        proxy: Proxy,
+        protocol: Protocol?,
+    ) {
+        printer.printlnWithStackTrace("connected to $inetSocketAddress")
+    }
+    override fun connectFailed(
+        call: Call,
+        inetSocketAddress: InetSocketAddress,
+        proxy: Proxy,
+        protocol: Protocol?,
+        ioe: IOException,
```

Comment:
The reason I didn't is because it wasn't really useful. These errors propagate to the caller, which means these exceptions already get logged elsewhere at just verbosity 1 or 2.